lib/parameter_substitution/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class ParameterSubstitution
-  VERSION = "1.2.0"
+  VERSION = "2.0.0"
```

Comment:
We can discuss in the slack thread where you initially asked about this as well

Would it be better to leave this unchanged if we don't plan to release the new version? It could be confusing for the next person if they come in wanting to release a patch to have to back this down to 1.2.1.

Reply:
Would the next person who comes along just release this with their patch all absorbed into version 2.0.0? I could see how having the version already changed can be confusing so maybe updating the changelog is enough for transparency when a new release comes along. Hmmm I can be convinced either way 😅

Reply:
Yeah I'm not sure either, this is uncharted territory for me. I'll move the question to slack.

Reply:
One could argue that the removal of Rails 4 support is a non-breaking minor change since (1) Rails 4 has been EOL for a very long time and no longer getting updated and (2) the newly added support wouldn't work on Rails 4 anyway. This would just mean that Rails4 users would need to lock to version `1.2.0` which won't get any more updates anyway.